api/api_full.go

```diff
@@ -584,6 +586,8 @@ type FullNode interface {
 	StateVMCirculatingSupplyInternal(context.Context, types.TipSetKey) (CirculatingSupply, error) //perm:read
 	// StateNetworkVersion returns the network version at the given tipset
 	StateNetworkVersion(context.Context, types.TipSetKey) (apitypes.NetworkVersion, error) //perm:read
+	// StateActorCids returns the CIDs of all the actors for the given network version
```

Comment:
Can we call it `StateActorCodeCIDs`?

And helptext could say `of all the builtin actors`